python/src/mas/cli/install/app.py

```diff
@@ -1143,6 +1137,10 @@ def nonInteractiveMode(self) -> None:
             self.validateCatalogSource()
             self.licensePrompt()
 
+        # Version before 9.1 cannot have empty components
```

Comment:
as part of changing the default to remove health from Manage ...  (the reason we defaulted to health being installed was to make it easy for the predict usecase mainly), now the default is "", I feel like we need a variation of code like we have here that is saying "if predict is also being installed then make sure `"health" in self.getParam("mas_appws_components")` ... this seems like the right point to have that check too?  It's happening after all the apps have been configured, so maybe it's as simple as adding another `if` after this one?